src/Generators/TestsGenerator.php

```diff
@@ -54,7 +54,11 @@ protected function getInserts()
     {
         $arrayModels = [$this->model];
 
-        if ($this->classExists('models', 'User') && $this->isFactoryExists('User')) {
+        if (
+            $this->classExists('models', 'User')
+            && $this->isFactoryExists('User')
+            && $this->isMethodExists('User', 'getFields')
```

Comment:
let's move it to the separate method `canGenerateUserData()`

```
return $this->classExists('models', 'User')
    && $this->isFactoryExists('User')
    && $this->isMethodExists('User', 'getFields')
```

Reply:
Done